plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -388,6 +375,25 @@ class NewCreateServiceModal extends Component {
     const {serviceFormErrors, serviceSpec} = this.state;
     let validationErrors = [];
 
+    const APP_VALIDATORS = Hooks.applyFilter(
```

Comment:
Please use the filter to get a list of additional filter and append them as discussed offline.